apps/content/services/tafsir.py

```
@@ -0,0 +1,127 @@
+
+    def create_tafsir(
+        self,
+        *,
+        publisher_id: int,
+        name_ar: str,
+        name_en: str,
+        description_ar: str,
+        description_en: str,
+        long_description_ar: str,
+        long_description_en: str,
+        license: str,
```

Ruff (0.15.9)

[error] Function argument `license` is shadowing a Python builtin

(A002)